spec/services/invoices/add_on_service_spec.rb

```diff
@@ -54,28 +54,28 @@
       end.to have_enqueued_job(SendWebhookJob)
     end
 
-    it 'does not enqueue an SendEmailJob' do
+    it 'enqueue an GeneratePdfAndNotifyJob with email false' do
       expect do
         invoice_service.create
-      end.not_to have_enqueued_job(SendEmailJob)
+      end.to have_enqueued_job(Invoices::GeneratePdfAndNotifyJob).with(hash_including(email: false))
```

Comment:
I have seen none of the tests that expects the new `GeneratePdfAndNotifyJob` being queued checks the invoice is passed as an argument. Would you consider to add the invoice argument expectation at least in the happy path?

Reply:
Thank you for the suggestion. The primary goal of our test is to verify that the GeneratePdfAndNotifyJob is enqueued with the correct email flag. I see the importance of passing the correct invoice as an argument; however, accessing the invoice within the current test setup is not straightforward.